src/main/java/com/dynatrace/prototype/payloadHandler/SlackPayloadHandler.java

```diff
@@ -0,0 +1,43 @@
+package com.dynatrace.prototype.payloadHandler;
+
+import com.slack.api.Slack;
+import com.slack.api.methods.SlackApiException;
+import com.slack.api.methods.request.chat.ChatPostMessageRequest;
+import com.slack.api.methods.response.chat.ChatPostMessageResponse;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.io.IOException;
+
+@ApplicationScoped
+public class SlackPayloadHandler implements PayloadHandler {
+    private static final String ENV_SLACK_TOKEN = "SLACK_TOKEN";
+    private static final String ENV_SLACK_CHANNEL = "SLACK_CHANNEL";
+
+    @Override
+    public boolean sendPayload(String payload) {
+        boolean messageSent = false;
```

Comment:
I would consider the case of an empty payload still a successful action. Only in the case of an exception or error on the slack side, I would return `false`

```suggestion
        boolean error = false;
```

Reply:
If the payload is empty, Slack will throw an exception.